pkgs/development/compilers/graalvm/default.nix

```diff
@@ -403,7 +414,7 @@ in rec {
       description = "High-Performance Polyglot VM";
       license = licenses.gpl2;
       maintainers = with maintainers; [ volth hlolli ];
-      platforms = [ "x86_64-linux" ];
```

Comment:
I have not tested yet on "aarch64-linux", but it suppose to work.
Enabled for ofborg to do the aarch64 build too